### DISCUSSION TITLE
Retain Custom Lore when Placing/Breaking

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#723] [__Dismissed__] [answered]
Hey!

I'm not sure if this is possible or if it's better suited for a plugin but I was wondering if there would be a way to add an option to retain custom lore when placing/breaking a block. Would love to make some trophies and other items that people are free to place down without them 'breaking' (deleting the lore)

Thanks!

**Comment by StrangeOne101** [comment, marked as answer]
This is only possible for Block Entities, such as furnaces, chests, shulker boxes, banners, etc. Normal blocks have no way to store the data.
